### ISSUE TITLE
Support mutagen sync on with WSL 2 using Lando events.

### ISSUE THREAD

**Issue by ayalon** [lando/lando#2541]
Lando combined with the new Windows Subsystem Linux 2 and Docker is amazing. I'm pretty sure this will quickly become the new star of the art DEV environment. As long as your files live inside the inside Unix distribution, you have native file system speed.

It is even faster than the current `cached` and `delegated` docker mount implementation of OSX. We could be early adopters and onboard all the potential Windows 10 developers to use Lando.

Microsoft added the` \\wsl$\<Distro Name>` file share, but as soon as your project has more than 100 files, it's unusable with PHPStorm or any other IDE. The file system performance of mounted volumes on WSL 2 is even 10x slower than on WSL 1.[ A lot of discussions and rants about this topic have been on on github.com](https://github.com/microsoft/WSL/issues/4197). 

The solution to this issue is mutagen.io sync. With this Open-Source tool, your files life inside your Ubuntu distro. Then you can two-way sync the files back to an NTFS folder and edit them on Windows.

Currently I have to manually start the mutagen sync process after `lando start` and terminate ist after `lando stop`. I did not find a way to automate this process, because Lando is installed on the Linux Distro and mutagen has to be started from a PowerShell on Windows ([see my blogpost with detailed instructions](https://www.liip.ch/en/blog/setup-drupal-and-lando-with-wsl2-on-windows).)

With the help of some experts, I'm pretty sure, we can extends Lando to support this steps out-of-the-box. I'm very much interested in ideas / inputs / examples how to do that.

**Comment by dustinleblanc** [comment]
Hey @ayalon!

Interesting setup, when I was heavily using WSL2 I was making use of VS Code because of the annoyance of doing PHPStorm setup with WSL2 so I feel the pain.

Though the results folks are having with WSL2 / Lando are really good so far, as a project, we don't really officially support installation via these means on Windows yet. I agree with you though that the results I and others have had are pretty stunning, so its likely going to become the best way to do things in Windows land.

This leaves us with a quandry though, we're not likely to code specific stuff into Lando to handle a workflow that isn't yet deemed the default way to do things on Windows, and since I switched my machine over to Fedora 32, we don't even have a maintainer regularly using Windows at the moment.

Other folks have also documented ways to install PHPStorm within the WSL2 distro and start an x server from there to use the IDE, so I am not 100% sure that wouldn't be preferred here.

In summary, worth considering, but I am going to ice box this for now as we won't be tackling it soon, but I'd certainly love to get more feedback from the rest of the Windows users on what their experience has been and what they think.

Thanks!
